src/rtc/schemas/rtc_s1.yaml

```diff
@@ -43,6 +43,16 @@ runconfig:
             #     {output_dir}/{burst_id}/{product_id}_v{product_version}{suffix}.{ext}
             # If option `save_mosaics` is set, output mosaics are saved to:
             #     {output_dir}/{product_id}_v{product_version}{suffix}.{ext}
+            #
+            # If the `product_id` contains the substring "_{burst_id}", the
+            # substring will be substituted by either:
+            #     - "_" followed by the burst ID, if the product is a burst; or
+            #     - An empty string, if the product is a mosaic.
+            #
+            # For example, the `product_id` = `RTC-S1_{burst_id}_S1B` will become
+            # `RTC-S1_069-147170-IW1_S1B` for the burst t069-147170-IW1; and it
```

Comment:
Just want to check if we can omit the letter `t` in the burst_id part of the product ID.
I mean, why `RTC-S1_069-147170-IW1_S1B` instead of  `RTC-S1_T069-147170-IW1_S1B`?

Reply:
We had a meeting about it, in that meeting it was decided to remove the 't' and also to use dashes rather than underbars. We can always change it later.